resources/views/editor/filelist.blade.php

```diff
@@ -1,8 +1,8 @@
 <table class="table table-striped table-hover">
     <thead>
         <tr>
-          <th>Filename</th>
-          <th>Type</th>
+          <th>Project Name</th>
```

Comment:
nah, the first rows were for debugging. name, type, desc, creator, timestamps should be fine.